anyway/widgets/all_locations_widgets/all_locations_widget.py

```diff
@@ -5,13 +5,13 @@
 
 
 class AllLocationsWidget(Widget):
-    def __init__(self, request_params: RequestParams, name: str):
+    def __init__(self, request_params: RequestParams):
```

Comment:
Consider adding "widget_digest" calc to here?

Reply:
I do not follow. It is now in `Widget`, what will we gain by adding it to lower classes?

Reply:
Moved to DM

Reply:
All invocation of `calc_widget_digest()` were replaced with a single invocation in `register` decorator.